sormas-app/app/src/main/res/layout/dialog_immunization_overlaps_layout.xml

```diff
@@ -0,0 +1,151 @@
+  -->
+
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/base_layout"
+    android:descendantFocusability="beforeDescendants"
+    tools:ignore="contentDescription">
+
+    <data>
+
+        <import type="de.symeda.sormas.app.component.controls.ControlButtonType" />
+
+        <variable
+            name="data"
+            type="de.symeda.sormas.app.immunization.edit.ImmunizationOverlapsDto" />
+
+        <import type="de.symeda.sormas.api.i18n.I18nProperties" />
+
+        <import type="de.symeda.sormas.api.i18n.Strings" />
+
+        <variable
+            name="dismissCallback"
+            type="de.symeda.sormas.app.util.Callback" />
+
+    </data>
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ScrollView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginTop="4dp"
+            android:layout_marginBottom="64dp">
+
+            <LinearLayout
+                android:id="@+id/main_content"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                android:paddingLeft="@dimen/rowListItemHorizontalMargin"
+                android:paddingRight="@dimen/rowListItemHorizontalMargin">
+
+                <TextView
+                    android:id="@+id/immunization_infoMessage"
+                    style="@style/SubHeadingStyle"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@{String.format(I18nProperties.getString(Strings.infoSimilarImmunization), data.getDisease().toString())}"
+                    android:textSize="@dimen/defaultControlTextSize" />
+
+                <TextView
+                    android:id="@+id/immunization_infoImmunizationPeriod"
+                    style="@style/SubHeadingDescriptionStyle"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@{I18nProperties.getString(Strings.infoImmunizationPeriod)}"
+                    android:textSize="@dimen/defaultControlTextSize" />
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal">
+
+                    <de.symeda.sormas.app.component.controls.ControlDateField
+                        android:id="@+id/immunization_startDate"
+                        style="@style/ControlFirstOfTwoColumnsStyle"
+                        app:value="@={data.startDate}" />
```

Comment:
Remove the = signs from the `app:value` parameters and turn the fields into ControlTextReadFields because they should not be editable